feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -432,8 +431,16 @@ fun ZoomScaleText(zoomScale: Float) {
         modifier = Modifier
             .alpha(contentAlpha.value)
             .testTag(ZOOM_RATIO_TAG),
-        text = "%.1fx".format(zoomScale),
-        fontSize = 20.sp
+        text = "%.1fx".format(zoomScale)
```

Comment:
Is this `%.1fx.format(zoomScale)` supposed to be replaced with formatting strings?
https://developer.android.com/guide/topics/resources/string-resource#formatting-strings

Reply:
Technically this is correct. It should be a localized string.

Reply:
Aha, thank you.